src/SimplCommerce.WebHost/appsettings.json

```diff
@@ -1,6 +1,6 @@
 ﻿{
   "ConnectionStrings": {
-    "DefaultConnection": "Server=.\\sqlexpress;Database=SimplCommerce;uid=sa;pwd=sa;Trusted_Connection=True;MultipleActiveResultSets=true"
+    "DefaultConnection": "Server=(localdb)\\MSSQLLocalDB;Database=SimplCommerce;uid=sa;pwd=sa;Trusted_Connection=True;MultipleActiveResultSets=true"
```

Comment:
@MaherJendoubi thanks, but we also need to support PostgreSQL. Could you please help to make it? or I will do